CHANGELOG.next.toml

```diff
@@ -58,3 +58,41 @@ message = "Update urlencoding crate to v2.1.0"
 references = ["smithy-rs#1301"]
 meta = { "breaking" = false, "tada" = false, "bug" = false }
 author = "benesch"
+
+[[smithy-rs]]
```

Comment:
This needs to be duplicated in the SDK changelog with `[[aws-sdk-rust]]` since it will impact SDK customers as well.

I want to see the codegen diff as part of reviewing this, so I'll merge the latest from main and just make this change as part of the merge conflict resolution.